packages/plugin-core/src/constants.ts

```diff
@@ -498,6 +498,10 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: DendronContext.PLUGIN_ACTIVE,
   },
+  GOTO_BACKLINK: {
```

Comment:
can we hide this from command palette?

Reply:
good catch, forgot how the commands manifest worked haha